classes/Link.php

```diff
@@ -81,6 +86,8 @@ public function __construct($protocolLink = null, $protocolContent = null)
      * @param int $idPicture ID of the picture to delete
      *
      * @return string
+     *
+     * @throws PrestaShopException
```

Comment:
Why adding this, this method doesn't throw PrestaShopException :thinking:

Reply:
PHPStorm inspection ;)

Reply:
Due to usage of Configuration::get() can return PrestaShopException on call Db::executeS

Reply:
I will update Pull Request according your feedback

Reply:
Phpstorm :vomiting_face: